File: apps/fogg/src/trip.go
package main

import (
	"fogg/src/pump/into"
	"fogg/src/type"
)

func trip() []types.Trip {
	return []types.Trip{
		{
			Address: into.StringToC("/version"),
			Payload: into.StringToC("{}"),
			Options: into.StringToC("into:some-id"),
		},
		{
			Address: into.StringToC("/storage"),
			Payload: into.StringToC("{}"),
			Options: into.StringToC("into:some-other-id"),
		},
	}
}

File: apps/rata/src/trip.rs
use crate::r#type::Trip;

pub fn trip() -> [Trip; 2] {
    [
        Trip {
            address: c"/version".as_ptr(),
            payload: cr#"{}"#.as_ptr(),
            options: c"into:some-id".as_ptr(),
        },
        Trip {
            address: c"/help".as_ptr(),
            payload: cr#"{}"#.as_ptr(),
            options: c"into:some-other-id".as_ptr(),
        },
    ]
}
